### PR TITLE
scripts: Update syncval stage and access masks snippets

### PR DESCRIPTION
**Pull request by jeremyg-lunarg**
The only change since 1.2.185 is the addition of VK_ACCESS_2_SHADER_BINDING_TABLE_READ_BIT_KHR.

### CODE REVIEW

File: layers/sync_vuid_maps.cpp [hunk 1]
```diff
@@ -364,6 +364,15 @@ static const std::map<VkAccessFlags2KHR, std::array<Entry, 6>> kAccessMask2Commo
          {Key(Struct::VkImageMemoryBarrier2, Field::srcAccessMask), "VUID-VkImageMemoryBarrier2-srcAccessMask-03906"},
          {Key(Struct::VkImageMemoryBarrier2, Field::dstAccessMask), "VUID-VkImageMemoryBarrier2-dstAccessMask-03906"},
      }}},
+    {VK_ACCESS_2_SHADER_BINDING_TABLE_READ_BIT_KHR,
+     {{
+         {Key(Struct::VkMemoryBarrier2, Field::srcAccessMask), "UNASSIGNED-VkMemoryBarrier2-srcAccessMask-SHADER_BINDING_TABLE_READ"},
```
Comment (jeremyg-lunarg) [inline layers/sync_vuid_maps.cpp:369]: A khronos gitlab issue has been filed asking for a VUID to go here.

**Review by Tony-LunarG — APPROVED** [review]
LGTM and fixes assert I was seeing

**Review by jzulauf-lunarg — APPROVED** [review]
LGTM

**Comment by Rua** [comment]
I noticed that the changes from this PR were removed again in 825dd8a8902b1005434016951f636f759d4bdcfb. Why?

File: layers/sync_vuid_maps.cpp (hunk 1 above)
Comment (arno-lunarg) [reply layers/sync_vuid_maps.cpp:369]: It has been added as `VUID-{barrier}-{accessMask}-07272`. I am going to make a PR.

**Pull request merged**